### DISCUSSION TITLE
Will latest MissionControl work with atmosphere 1.5.5?

### DISCUSSION THREAD

**Discussion by Monchhichis** [ndeadly/MissionControl#747] [General]
I have a hacked unpatched Switch with BASE firmware - 16.0.3 AMS - 1.5.5

I see the Mission Control has requirement of up to date version of Atmosphere but I don't know how to update it to 1.6.0...

Will installing latst Mission Control on AMS 1.5.5 cause the system to crash? If so, can I install an older version of Mission Control (that supports atmosphere 1.5.5) instead for PS4 controllers?

**Comment by ndeadly** [comment]
The up-to-date requirement only applies to latest firmware. Crashing on boot only occurs when you have an updated firmware and atmosphere, but an outdated mission control version. This typically happens around the time of a new update. The other way around, where you have a newer version of mission control is fine - they are all backwards compatible.

When in doubt, just try it and see. The worst thing that can happen is a crash on boot. This can easily be resolved by removing the  files you added.
